src/shared/Classes.ts

```diff
@@ -100,3 +100,52 @@ export class Package {
 
   getPackage(t: Locale, exhibitorPackage: string) {}
 }
+
+export class Exhibitor {
```

Comment:
Not that important but what is the use of this class? If we just return some shape from the query for exhibitors, typescript knows what to do

Reply:
I thought so too, but TRPC seems to have an issue with it (see image). 
Also, the logos had to be converted to `base64` server-side due to what seems to be a bug.

<img width="438" alt="image" src="https://github.com/datasektionen/ddagen/assets/66337956/361a7176-d3f0-41da-a7d1-aa027b306a1a">

Reply:
I think that is because the logos are just binary data, which can be solved by converting to base64 as you do now.